File: string/remove-palindromic-subsequences/remove-palindromic-subsequences.go
package removepalindromicsubsequences

func removePalindromeSub(s string) int {
	i := 0
	j := len(s) - 1
	for i < j {
		if s[i] == s[j] {
			i++
			j--
		} else {
			return 2
		}
	}

	return 1
}

File: string/remove-palindromic-subsequences/remove-palindromic-subsequences.ts
function removePalindromeSub(s: string): number {
    let i = 0;
    let j = s.length - 1;

    while (i < j) {
        if (s[i] == s[j]) {
            i++;
            j--;
        } else {
            return 2;
        }
    }
    return 1;
};
